restservices/src/main/java/com/hydra/restservices/controllers/MainController.java

```diff
@@ -1,29 +1,37 @@
 package com.hydra.restservices.controllers;
 
-import org.springframework.stereotype.Controller;
+import com.hydra.restservices.repository.ExchangeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import com.hydra.restservices.services.ExchangeService;
 
-@Controller
-@RequestMapping("/exchange")
+@RestController
+@RequestMapping(value = "/exchange")
 public class MainController {
 
-    @RequestMapping(value = "/", method = RequestMethod.GET)
+    @Autowired
+    private ExchangeService exchangeService;
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
     public Object getAllExchanges() {
-        return null;
+        System.out.println(" Someone hit the API");
+        System.out.println(exchangeService.getAllExchanges());
+        return exchangeService.getAllExchanges();
     }
 
     @RequestMapping(value = "/{exchangeId}", method = RequestMethod.GET)
-    public Object getCoinsInAnExchange(@PathVariable String exchangeId) {
-        return null;
+    public Object getCoinsInExchange(@PathVariable String exchangeId) {
+        System.out.println(" Someone hit the API");
```

Comment:
Same here

Reply:
Log the message with precise messages please